src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -209,37 +209,23 @@ services:
       $countryChoices: '@=service("prestashop.core.form.choice_provider.country_by_iso_code").getChoices()'
 
   PrestaShopBundle\Form\Admin\Improve\Payment\Preferences\AbstractPaymentModuleRestrictionsType:
```

Comment:
Remove `tags:` section as well (for all types you've edited)

Reply:
Removed them